pipes/WDL/tasks/tasks_16S_amplicon.wdl

```diff
@@ -160,59 +163,66 @@ task merge_paired_ends {
-        String  reads_basename = basename(reads_qza, '.qza')
-        Int?    trim_length_var = 300     
+        String  joined_end_basename = basename(joined_end_outfile, '.qza')
+        Int     trim_length_var = 300
+        Int     memory_mb = 2000
+        Int     cpu = 1
+        Int     disk_size_gb = ceil(2*size(joined_end_outfile, "GiB")) + 5
         String  docker = "quay.io/qiime2/core:2022.8"
     }
-    command <<< 
-            set -ex -o pipefail
-            CONDA_ENV_NAME=$(conda info --envs -q | awk -F" " '/qiime.*/{ print $1 }')
-            conda activate ${CONDA_ENV_NAME}
-            ##activated conda environments 
+        command <<< 
+        set -ex -o pipefail
+        # obtain the name of the qiime conda environment in the container; flexible about version
+        CONDA_ENV_NAME=$(conda info --envs -q | awk -F" " '/qiime.*/{ print $1 }')
+        # activate the qiime conda environment
+        # seemingly necessary because of:
+        # https://github.com/chanzuckerberg/miniwdl/issues/603
+        conda activate ${CONDA_ENV_NAME}
             qiime deblur denoise-16S \
-            --i-demultiplexed-seqs "~{reads_basename}_demux_joined.qza" \ 
-            ~{"--p-trim-length" + trim_length_var} \ 
+            --i-demultiplexed-seqs ~{joined_end_outfile} \
+            ~{"--p-trim-length " + trim_length_var} \
             --p-sample-stats \
-            --o-representative-sequences "~{reads_basename}_rep-seqs.qza" \
-            --o-table "~{reads_basename}_table.qza"
+            --o-representative-sequences "~{joined_end_basename}_rep_seqs.qza" \
+            --o-table "~{joined_end_basename}_table.qza" \
+            --o-stats "~{joined_end_basename}_stats.qza"
             
             #Generate feature table- give you the number of features per sample 
             qiime feature-table summarize \
-            --i-table  "~{reads_basename}_table.qza" \
-            --o-visualization   "~{reads_basename}_table.qzv"
+            --i-table  "~{joined_end_basename}_table.qza" \
+            --o-visualization   "~{joined_end_basename}_table.qzv"
             #Generate visualization of deblur stats
             qiime deblur visualize-stats \
-            --i-deblur-stats "~{reads_basename}_deblur_stats.qza" \
-            --o-visualization "~{reads_basename}_deblur_stats.qzv"
-            >>>
+            --i-deblur-stats "~{joined_end_basename}_stats.qza" \
+            --o-visualization "~{joined_end_basename}_stats.qzv"
+        >>>
     output {
-        #how many output files do i need
-        File rep-seqs_outfile = "~{reads_basename}_rep-seqs.qza"
-        File rep-table_outfile = "~{reads_basename}_table.qza"
-        File feature-table = "~{reads_basename}_table.qzv"
-        File visualize-stats = "~{reads_basename}_deblur_stats.qzv"
+        File rep_seqs_outfile = "~{joined_end_basename}_rep_seqs.qza"
+        File rep_table_outfile = "~{joined_end_basename}_table.qza"
```

Comment:
Let's not use `outfile` to describe things that are already in an `output` block (everyone knows it's an output)
```suggestion
        File representative_seqs_qza = "~{joined_end_basename}_rep_seqs.qza"
        File representative_table_qza = "~{joined_end_basename}_table.qza"
```